Week3/homework/5-step3.js

```diff
@@ -1,2 +1,17 @@
 'use strict';
-// paste your freeCodeCamp solutions in here
+function multiplyAll(arr) {
+  var product = 1;
+  for (var i = 0; i < arr.length; i++) {
+    for (var j = 0; j < arr[i].length; j++) {
+      console.log(arr[i][j]);
+      product = product * arr[i][j];
+       console.log(product * arr[i][j]);
+    }
+  }
+  return product;
+}
+
+// Modify values below to test your code
+multiplyAll([[1], [2], [3]]);
+multiplyAll([[1, 2], [3, 4], [5, 6, 7]]);
+multiplyAll([[5, 1], [0, 2, 4, 0.5], [3, 9]]);
```

Comment:
This third array has issues because it has three sub-arrays of unequal length, respectively 2, 3 and 2. Furthermore, because one of the array elements is zero, the final product will also be zero.